<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BancoSecureBit</string>
  </property>
  <property name="styleSheet">
   <string notr="true">.QWidget{background-color: rgb(25,25, 40);}</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="btnRegistrarTransferencias"/>
    <addaction name="btnReportarTransferencias"/>
    <addaction name="btnHistorialTransferencias"/>
   </widget>
   <addaction name="menuArchivo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="btnRegistrarTransferencias">
   <property name="text">
    <string>Registrar Transferencias</string>
   </property>
  </action>
  <action name="btnReportarTransferencias">
   <property name="text">
    <string>Reportar Transferencias</string>
   </property>
  </action>
  <action name="btnHistorialTransferencias">
   <property name="text">
    <string>Historial de Transferencias</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
